### ISSUE TITLE
Bump outdated peer dependencies

### ISSUE THREAD

**Issue by zackdotcomputer** [electron-react-boilerplate/eslint-config-erb#136]
Would appreciate an update to these peer dependencies to remove these warnings if one is at the latest release for each of these packages:

```
warning " > eslint-config-erb@2.0.0" has incorrect peer dependency "eslint-config-airbnb-typescript@^8.0.2".
warning " > eslint-config-erb@2.0.0" has incorrect peer dependency "eslint-config-prettier@^6.11.0".
warning " > eslint-config-erb@2.0.0" has incorrect peer dependency "eslint-import-resolver-webpack@^0.12.2".
warning " > eslint-config-erb@2.0.0" has incorrect peer dependency "eslint-plugin-jest@^23.13.2".
warning " > eslint-config-erb@2.0.0" has incorrect peer dependency "eslint-plugin-jsx-a11y@6.2.3".
warning " > eslint-config-erb@2.0.0" has incorrect peer dependency "react@^16.13.1".
```

**Comment by amilajack** [comment]
I'll get to this soon, thanks for filing this

**Comment by amilajack** [comment]
fixed in eslint-config-erb@4.0.5
